packages/react-native-reanimated/Common/cpp/SharedItems/Shareables.h

```diff
@@ -183,16 +183,20 @@ class ShareableObject : public Shareable {
  public:
   ShareableObject(jsi::Runtime &rt, const jsi::Object &object);
 
+#if defined(USE_HERMES) || REACT_NATIVE_MINOR_VERSION >= 74
```

Comment:
Let's add
```
#define SUPPORTS_NATIVE_STATE (defined(USE_HERMES) || REACT_NATIVE_MINOR_VERSION >= 74)
```
in `Shareable.h`
and use `#if SUPPORTS_NATIVE_STATE` everywhere else.

Reply:
+1 for this change

Apart from it, the rest is ok 👍

Reply:
You got it @tomekzaw @piaskowyk 👍